### DISCUSSION TITLE
How to change input colors?

### DISCUSSION THREAD

**Discussion by MrDowntempo** [starship/starship#5351] [Q&A] [answered]
Does starship have control over the color of the text you enter at the prompt? Or is that handled by whatever shell/terminal emulator you're also using? I couldn't find any info in the docs, but I probably wasn't looking for the right terminology.

**Comment by mickimnet** [comment, marked as answer]
Nope, _Starship_ only has control over the prompt, and not on the input line (command line) itself. That is something your shell has control over.
